src/ontology/uberon-edit.obo

```diff
@@ -10362,6 +10362,7 @@ property_value: taxon_notes "Most vertebrates have some form of duct to transfer
 id: UBERON:0001001
 name: chitin-based cuticle
 def: "A cuticular covering that is composed primarily of chitin. The main structural component of arthropod cuticle is a polysaccharide, chitin, composed of N-acetylglucosamine units, together with proteins and lipids[WP]." [https://orcid.org/0000-0002-6601-2165, https://sourceforge.net/tracker/index.php?func=detail&aid=3348965&group_id=36855&atid=440764, Wikipedia:Arthropod_exoskeleton, Wikipedia:Cuticle#Invertebrate_zoology]
+comment: "TODO - split from exoskeleton? Also: See above sf item to see if this belongs in GO" xsd:string
```

Comment:
In OBO format the `comment` tag expects a **unquoted** string (contrary to `def`) without a type qualifier. Using quotes followed by a `xsd:string` type qualifier results in the parser interpreting the value of the comment to be **literally**

```
"TODO - split from exoskeleton? Also: See above sf item to see if this belongs in GO" xsd:string
```

as can be seen when converting the file to functional syntax:

```
AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/UBERON_0001001> "\"TODO - split from exoskeleton? Also: See above sf item to see if this belongs in GO\" xsd:string")
```

Reply:
Good, I found other examples not part of the `editor notes` change.